modules/Book.js

```diff
@@ -0,0 +1,70 @@
+import printErrorMsg from './errorMsg.js';
+
+let books = JSON.parse(localStorage.getItem('books'));
+
+export default class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
```

Comment:
- [ ] Kindly refactor your function to use ES6 syntax (arrow function)

Reply:
Resolved. Thanks